demos/python_demos/single_human_pose_estimation_demo/README.md

```diff
@@ -0,0 +1,49 @@
+# Single Human Pose Estimation Demo (top-down pipeline)
+
+This demo showcases top-down pipeline for human pose estimation on video or image. The task is to predict bboxes for every person on frame and then to predict a pose for every detected person. The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wrists, hips, knees, and ankles.
```

Comment:
What does top-down mean in this context?

Reply:
Top-down mean that first, we detect all people in image, then crop image for each person and estimate their pose. @IRDonch